lib/runner/Configuration.js

```diff
@@ -110,9 +110,17 @@ module.exports = class Configuration {
             }
         }
 
-        this.configurationJSON.jest = jestConfig;
         this.overrideConfigurationWithEnvVariables();
         this.overrideConfigurationWithCli(cliOverrides);
+
+        const jestReporters = this.configurationJSON.reporters;
+        if (jestReporters) {
+            this.setUpReporters(jestConfig, jestReporters);
+        } else {
+            jestConfig.reporters = ["default", "jest-html-reporters"];
+        }
+
+        this.configurationJSON.jest = jestConfig;
     }
```

Comment:
this one is a little tricky because the variables we set on jest are not the same that the user uses. 
console is translate to default and html is translated to jest-html-reporters if we set them before overrides, then those would be the final values. 
Since override modify configurationJSON, i evaluate the reporters property after the override. You can see on the tests that it works when the override parameters are set